daemon/src/rollover/maker.rs

```diff
@@ -236,6 +239,7 @@ impl Actor {
                 }
             },
         );
+        self.protocol_tasks.insert(order_id, tasks);
```

Comment:
What happens with the existing `task` if it is being overwritten by another one?

Reply:
it gets dropped. that's why it's not a general solution, and not safe to roll out in other cases (e.g. ping, where we periodically add them). Here due to our business rules we can't perform a decision twice for the same rollover. Technically, if the future execution would take over an hour, we would still overwrite it - but at that point, it would probably be a good thing to do anyway :-)

Reply:
Thanks for the clarification